packit/config/common_package_config.py

```diff
@@ -329,6 +332,9 @@ def __init__(
             allowed_pr_authors if allowed_pr_authors is not None else ["packit"]
         )
         self.allowed_committers = allowed_committers or []
+        self.allowed_builders = (
+            allowed_builders if allowed_builders is not None else ["packit"]
```

Comment:
we will need to edit this for our packages to include `packit-stg`, but at least we can test it